src/test/java/org/rulez/demokracia/PDEngine/VoteCreationTest.java

```diff
@@ -110,4 +111,340 @@ public void create_creates_a_vote_with_adminKey() {
 		assertEquals(VoteRegistry.getByKey(adminInfo.adminKey).adminKey, adminInfo.adminKey);
 	}
 
+// voteName validation tests
+
+	@Test
+	public void votename_contains_wrong_characters_input1() throws ReportedException {
```

Comment:
The annotations for the behaviour on the class refer to another behaviour. Maybe the best approach would be to create a new test class for this behaviour, and annotate it.